commit ed96e734be83e666ea9fd16cc32fc50bb5e665d4
Author:     Holger Roth <6304754+holgerroth@users.noreply.github.com>
AuthorDate: Thu Apr 28 18:18:32 2022 -0400
Commit:     GitHub <noreply@github.com>
CommitDate: Thu Apr 28 15:18:32 2022 -0700

    Add a white space to log info when deploying an application (#459)
---
 nvflare/private/fed/server/job_runner.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/nvflare/private/fed/server/job_runner.py b/nvflare/private/fed/server/job_runner.py
index 280c0194..bcb14c62 100644
--- a/nvflare/private/fed/server/job_runner.py
+++ b/nvflare/private/fed/server/job_runner.py
@@ -68,7 +68,7 @@ class JobRunner(FLComponent):
                 if p == "server":
                     success = deploy_app(app_name=app_name, site_name="server", workspace=workspace, app_data=app_data)
                     self.log_info(
-                        fl_ctx, f"Application{app_name} deployed to the server for run:{run_number}", fire_event=False
+                        fl_ctx, f"Application {app_name} deployed to the server for run:{run_number}", fire_event=False
                     )
                     if not success:
                         raise RuntimeError("Failed to deploy the App to the server")
@@ -80,7 +80,7 @@ class JobRunner(FLComponent):
             display_sites = ",".join(client_sites)
             self.log_info(
                 fl_ctx,
-                f"Application{app_name} deployed to the clients: {display_sites} for run:{run_number}",
+                f"Application {app_name} deployed to the clients: {display_sites} for run:{run_number}",
                 fire_event=False,
             )
 
